src/Compilers/Core/Portable/Operations/OperationCloner.cs

```diff
@@ -527,7 +527,7 @@ public override IOperation VisitFlowCapture(IFlowCaptureOperation operation, obj
 
         public override IOperation VisitFlowCaptureReference(IFlowCaptureReferenceOperation operation, object argument)
         {
-            throw ExceptionUtilities.Unreachable;
+            return new FlowCaptureReference(operation.Id, operation.Syntax, operation.Type, constantValue: operation.ConstantValue);
```

Comment:
I think I'm missing where this is needed?

Reply:
> I think I'm missing where this is needed? 

See ```OperationCloner.CloneOperation(resource)``` call on line 2415 in ControlFlowGraphBuilder.cs

---
In reply to: [178670191](https://github.com/dotnet/roslyn/pull/25828#discussion_r178670191) [](ancestors = 178670191)

Reply:
Thanks.

---
In reply to: [178670593](https://github.com/dotnet/roslyn/pull/25828#discussion_r178670593) [](ancestors = 178670593,178670191)